sr/robot3/robot.py

```diff
@@ -411,25 +420,33 @@ def wait_start(self) -> None:
         def null_button_pressed() -> bool:
             return False
 
+        if not self._no_pb:
+            start_button_pressed = self.power_board._start_button
+        else:
+            # null out the start button function
+            start_button_pressed = null_button_pressed
+
```

Comment:
Nit: Flip condition to avoid negation

```suggestion
        if self._no_pb:
            # null out the start button function
            start_button_pressed = null_button_pressed
        else:
            start_button_pressed = self.power_board._start_button

```